web/packages/design/src/ThemeProvider/globals.js

```diff
@@ -67,6 +67,7 @@ const GlobalStyle = createGlobalStyle`
 
   // remove dotted Firefox outline
   button, a {
+    font-family: ${props => props.theme.font};
```

Comment:
this existing comment doesn't make much sense anymore if we are also changing the font now too. i think we need to rethink how we are applying this font, or update the comment (i prefer the first but you pick)